tests/cephfs/cephfs_bugs/test_large_omap.py

```diff
@@ -0,0 +1,98 @@
+import json
+import secrets
+import string
+import time
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83575630 - Bug 2164338 - Large Omap objects found in pool 'ocs-storagecluster-cephfilesystem-metadata'
+
+    Steps Performed:
+    1. Create a new filesystem
+    2. Mount the filesystem on a client
+    3. Set mds_bal_split_size to 1000
+    4. Touch 11000 file and create sanpshot
+    5. Initiate scrub on the filesystem
+    4. Check the ceph health for large omaps
+    5. Cleanup
+
+    Returns:
+
+    """
+    try:
+        tc = "CEPH-83575630"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fs_name = "cephfs_large_omap"
+        fs_util.create_fs(client1, fs_name)
+        fuse_mount_dir = "/mnt/fuse_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mount_dir,
+            new_client_hostname="admin",
+            extra_params=f" --client_fs {fs_name}",
+        )
+        client1.exec_command(
```

Comment:
Can you check the default value before changing it here?
and set it back to default once the test is completed.